clang/lib/CConv/CastPlacement.cpp

```diff
@@ -9,77 +9,55 @@
 // classes of CastPlacement.h
 //===----------------------------------------------------------------------===//
 
+#include <clang/Tooling/Refactoring/SourceCode.h>
```

Comment:
Nice -- glad we can use this. And glad you have cleared away the unnecessary `#include`s...